kapitan/secrets.py

```diff
@@ -54,36 +54,36 @@
 
 
 class GPGError(Exception):
-    "Generic GPG errors"
+    """Generic GPG errors"""
```

Comment:
Nit: Next time would be good to separate those or mention it in PR.

Reply:
missed these on the other MR